service/src/main/resources/api/service_openapi.yaml

```diff
@@ -1962,8 +1965,14 @@ components:
           $ref: "#/components/schemas/Name"
         description:
           type: string
+        bucketName:
+          description: Name of created bucket (not the resource name).
+          type: string
+        objectName:
+          description: Full path to the file in the gcs bucket.
+          type: string
         resourceAttributes:
-          description: attributes specific to GCS bucket object
+          description: (DEPRECATED) attributes specific to GCS bucket object
```

Comment:
yeah I couldn't figure out a clean way to change the API mostly because the attributes field is also used for other things like creating a referenced resources. They should be required for those cases so I'm hesitant to make those fields optional. 

2. I'm planning to make the change in CLI side and update the WSM version there after this goes in. Sounds good to just delete ResourceAttributes.